io/transmat/gs/gs_token.go

```diff
@@ -0,0 +1,77 @@
+package gs
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/storage/v1"
+)
+
+const EnvGsAccessToken = "GS_ACCESS_TOKEN"
+const EnvGsAccountFile = "GS_SERVICE_ACCOUNT_FILE"
+
+func GetAccessToken() (*oauth2.Token, error) {
+	var err error
+	var secrets []byte
+	token := getAccessTokenFromEnv()
+	if token != nil {
+		return token, nil
+	}
+	secrets, err = getServiceAccountSecretsFromEnv()
+	if err == nil {
+		token, err = createAccessTokenFromSecrets(secrets, []string{storage.DevstorageReadWriteScope})
+	}
+	return token, err
+}
+
+func getAccessTokenFromEnv() *oauth2.Token {
+	accessToken := os.Getenv(EnvGsAccessToken)
+	if accessToken != "" {
+		return &oauth2.Token{AccessToken: accessToken}
```

Comment:
Hm.  I wish there was a way to sanity-check the values here to make sure it smells like an oauth token -- It'd be nice to give a "token invalid" error right away without contacting a network service if someone does `export GS_ACCESS_TOKEN="asdflol"` -- but I don't see any way to do that in the library either ([the `Valid()` predicate is... uninspiring](https://github.com/golang/oauth2/blob/2baa8a1b9338cf13d9eeb27696d761155fa480be/token.go#L131)), so okay, I guess we'll do without. :hear_no_evil: